bedrock/mozorg/templates/mozorg/contentful-homepage.html

```diff
@@ -38,13 +38,16 @@
 <meta name="google-site-verification" content="U9a6gH32vLIykvntaDToj-ytYhlZ1AfAgVEKstixQIE">
 {% endblock %}
 
+{% set show_pocket_banner = switch('pocket-banner') and (LANG.startswith('en') or LANG == 'de') %}
```

Comment:
LANG conditions could be removed here? Leaving for consistency but I'm on the fence about it.